domains/auth/Auth.API/Unit.Tests/SwaggerTests.cs

```diff
@@ -2,7 +2,6 @@
 
 namespace Unit.Tests;
 
-[UsesVerify]
```

Comment:
Why has this been removed?

Reply:
Newest version of Verify, does not use this anymore. It is implicitly enabled by default, and can be invoked by just calling the api in the test